src/pages/LogInWithShortLivedAuthTokenPage.js

```diff
@@ -74,7 +74,7 @@ class LogInWithShortLivedAuthTokenPage extends Component {
     }
 
     render() {
-        // TODO: Not sure if we should redirect the user to the login page or just show the expiration screen or offline message
+        // redirect the user to the login page if there is a sudden disconnection
```

Comment:
I'm unsure what the exact edge case we're trying to handle is based on this comment. The user hits this page and in between downloading the JS bundle and executing it they somehow become offline? Can you expand on what exactly we are preventing and why we are doing this?

Reply:
I'm sorry I don't think I fully understood the previous comments. I don't think the desktop app needs to download the js bundle because it's an electron app and should be bundled with the app (IIRC).

>if the network is disconnected at this time

If they are disconnected then how can they access the page at all 🤔 

>Without this condition, the session expired ui will be displayed, if we think this page is also ok, then we can remove this condition.

I think it's fine but the message about the session being expired seems wrong?